lib/iris/fileformats/_pyke_rules/fc_rules_cf.krb

```diff
@@ -1018,6 +1017,7 @@ fc_extras
     import iris.exceptions
     import iris.std_names
     import iris.util
+    from iris._lazy_data import as_lazy_data
```

Comment:
Ah, I put this here because I'd seen examples where the private modules were imported last :wink: